docs/editor-script.zh-CN.md

```diff
@@ -78,9 +95,30 @@ export class Script1250228 extends Script {
 
 > 注意：不要写空函数，如果需要使用某个生命周期函数时再写，这样性能可以得到一定的提升。
 
-进入代码编辑界面之后，用户就可以根据自己的需求在提供的生命周期回调函数中实现自己的需求了。修改完毕后，按 `Ctrl+S` 保存右侧的预览面板会自动刷新。
+脚本的详细介绍见：[脚本组件](${docs}script-cn)
+
+当你修改完毕后，按 `Ctrl/CMD + S` 代码即可保存，右侧的实时预览区即可展现最新的效果。
+
+
+> 提示：Oasis的代码编辑器使用了vscode团队开源的monaco编辑器，编辑体验与快捷键与vscode基本一致
```

Comment:
Need spaces around the English words, and capitalize these.

Reply:
done